ui/helpers/utils/permission.js

```diff
@@ -50,7 +51,7 @@ function getSnapNameComponent(snapName) {
 }
 
 export const PERMISSION_DESCRIPTIONS = deepFreeze({
-  [RestrictedMethods.eth_accounts]: ({ t }) => ({
+  [Caip25EndowmentPermissionName]: ({ t }) => ({
```

Comment:
Assuming I'm understanding correctly we would still need to have a permission description for `eth_accounts` because it is used in the approvals still?

Reply:
![image](https://github.com/user-attachments/assets/e5828a59-0b64-4f3a-b69d-20a5278b0410)

Reply:
Thank you for catching this. I've resolved the issue here https://github.com/MetaMask/metamask-extension/pull/27847/commits/f460b5c24bd2e2a63d4cb1bc5c43ef2b1aba3901